hypervisor/arch/x86/guest/vmcs.c

```diff
@@ -230,9 +230,7 @@ static uint32_t check_vmx_ctrl(uint32_t msr, uint32_t ctrl_req)
 	ctrl |= vmx_msr_low;
 
 	if ((ctrl_req & ~ctrl) != 0U) {
-		pr_err("VMX ctrl 0x%x not fully enabled: "
-			"request 0x%x but get 0x%x\n",
-			msr, ctrl_req, ctrl);
+		pr_info("VMX ctrl 0x%x not fully enabled: request 0x%x but get 0x%x\n", msr, ctrl_req, ctrl);
```

Comment:
Question: are we "requesting" a specific value for VMX ctrl? When I scanned through the code, it looks like we're merely checking for specific features, not setting or requesting any. If true, then a better wording for this `pr_info` may be:
```suggestion
		pr_info("VMX ctrl 0x%x not fully enabled: checked for 0x%x but it is set to 0x%x\n", msr, ctrl_req, ctrl);
```

Reply:
here hints it is not a problem we use pr_info instead of pr_err,  and we have checked the essential VMX capabilities in "check_essential_vmx_caps"

change the log like this?  the ctrl value is from the read-only MSR
pr_info("VMX ctrl 0x%x not fully enabled: expected for 0x%x but get 0x%x\n", msr, ctrl_req, ctrl);

Reply:
I see, thanks. How about the following:
```
pr_info("VMX ctrl 0x%x not fully enabled: current capabilities are 0x%x (full capabilities are 0x%x)\n", msr, ctrl, ctrl_req);
```

Reply:
have update, pls check, thanks.